assets/data/cards_pt-br.json

```
@@ -29,6 +29,15 @@
       ],
       "description": "Costumamos chamar de ambiente o lugar ou os servidores onde o que foi desenvolvido fica disponível para testes, validações, pessoas usuárias finais utilizarem ou demais necessidades. Esse servidor seria como um computador. Pode ser um computador físico ou disponível na nuvem. Existem os ambientes local e dev (onde o desenvolvimento está acontecendo), homolog (ou ainda hom, homol, homologação - onde os testes costumam ser feitos)e produção (onde está disponível o sistema final, desenvolvido e testado, para pessoas usuárias)."
     },
+    {
+      "title": "Angular.js",
+      "tags": [
+        "Conceito",
+        "Front-end",
+        "Framework"
+      ],
+      "description": "Angular.js é um framework JavaScript criado pelo Google para desenvolver aplicações de página única (SPAs). Possui uma arquitetura robusta baseada em módulos e injeção de dependências. É muito usado por grandes corporações como Google e Microsoft"
```

Usei termos mais simples e adicionei uma explicação sobre SPAs para deixar mais claro para quem não está familiarizado com o conceito. O que acha?

```suggestion
      "description": "Angular.js é uma ferramenta desenvolvida pelo Google que facilita a criação de sites ou aplicativos que funcionam como uma única página (SPAs), ou seja, sem recarregar a página inteira a cada interação. Ele organiza o código de forma eficiente usando "módulos" e "injeção de dependências", o que ajuda a manter tudo mais fácil de gerenciar. Grandes empresas como Google e Microsoft utilizam essa tecnologia."
```